packages/mobile/src/screens/profile-screen/ProfileHeaderV2/SupportingTile.tsx

```diff
@@ -97,48 +108,58 @@ export const SupportingTile = (props: SupportingTileProps) => {
   }, [navigation, handle])
 
   const iconProps = {
-    height: spacing(3),
-    width: spacing(3),
-    marginRight: 6
+    height: spacing(3.75),
+    width: spacing(3.75)
   }
 
-  const supporterIcon = isTopRank ? (
-    <IconTrophy fill={secondary} {...iconProps} />
-  ) : (
-    <IconTip fill={neutralLight4} {...iconProps} />
-  )
-
   return user ? (
     <Tile style={[styles.root, style]} onPress={handlePress}>
       <ImageBackground
-        style={styles.supporterInfo}
+        style={styles.backgroundImage}
         source={{ uri: coverPhoto }}
       >
         <LinearGradient
-          colors={['#0000004D', '#0000001A']}
```

Comment:
Would love to learn how you made these changes with linear gradient sometime

Reply:
Just made it match the new figma:

```css
background: linear-gradient(180deg, rgba(0, 0, 0, 0.1) 50%, rgba(0, 0, 0, 0.3) 100%), url(.jpg);
```
Convert rgba to hex for each of those, and use the same angle. I believe the stops are defaulted how we want them, but if not I could add `locations=[0.5,1]`